praw/models/reddit/comment.py

```diff
@@ -243,7 +243,7 @@ def parent(self) -> Union[_Comment, Submission]:
            # `replies` is empty until the comment is refreshed
            print(parent.replies)  # Output: []
            parent.refresh()
-           print(parent.replies)  # Output is at least: [Comment(id='cklhv0f')]
+           print(parent.replies)  # Output is at least: [Comment(id="cklhv0f")]
```

Comment:
Output will technically use single-quotes.
```suggestion
           print(parent.replies)  # Output is at least: [Comment(id='cklhv0f')]
```